napari_aicsimageio/core.py

```diff
@@ -4,13 +4,12 @@
 from functools import partial
 from typing import Any, Callable, Dict, List, NamedTuple, Optional, Tuple, Union
 
-from pluggy import HookimplMarker
-
 import dask.array as da
 import numpy as np
 from aicsimageio import AICSImage, dask_utils, exceptions
 from aicsimageio.constants import Dimensions
 from aicsimageio.readers.reader import Reader
+from pluggy import HookimplMarker
```

Comment:
you can get rid of this right (along with the `napari_hook_implementation` below)? `napari_hook_implementation` isn't used in this module...

If you do need it though, might as well do `from napari_plugin_engine import HookImplMarker`

Reply:
Nice catch. Totally unused import.